### PR TITLE
Add requirements to participate

### PR DESCRIPTION
**Pull request by chadwhitacre**
This adds participation requirements to the homepage, based on @DuaneOBrien in Slack:

> [T]he working group would benefit from some clearer definition about what's required to participate. [W]hen I think about "should this person be added to the invite list or not" I have two primary considerations:
> - Are they working to refine models that help unlock funding for open source projects?
> - Are they doing this work agnostic of their own product development?

# New

Screenshot 2024-03-14 at 11 54 36 AM

# Old

Screenshot 2024-03-14 at 11 55 08 AM

### CODE REVIEW

**Review by bmuenzenmeyer — APPROVED** [review]

**Comment by hartwork** [comment]
I'm struggling to really grasp what "Are they doing this work agnostic of their own product development?" is supposed to mean. I'm not a native speaker of English but most people on the Internet are not. Maybe this needs an example or wording that reaches everyone.

**Comment by chadwhitacre** [comment]
The intention is that FOSS Funders is not a forum to pitch new OSS funding platforms (such as GitHub Sponsors, Open Collective, etc.).

@DuaneOBrien Can we find some language to clarify?

**Comment by hartwork** [comment]
> The intention is that FOSS Funders is not a forum to pitch new OSS funding platforms (such as GitHub Sponsors, Open Collective, etc.).

@chadwhitacre that makes good sense, thanks for the explanation.

**Comment by ShaneCurcuru** [comment]
Interesting: when I read "agnostic", I think of tech people in $BigCo who are looking to truly fund the FOSS ecosystem their company relies on for various software.

The "less-good" kind of participant, in my mind, would be the marketing type who is running a FOSSFund for marketing points, always funneling money into projects that directly make their own commercial software offering look good, help marketshare, whatever.

So I see a different likely misinterpretation than Chat explained (i.e. companies where the business itself is a funding model of some kind).

**Comment by DuaneOBrien** [comment]
@chadwhitacre is right in capturing the intention I was trying to get at - the working group is not a product focus group or a place for folks to come to get insights in how to develop their platforms. There are some interesting facets to that perspective that are worth discussing. 

For example, supposing GitHub decided to start a direct funding initiative where they were writing checks to projects on which GitHub had a dependency. In that case, I would find it reasonable to include someone involved in the decision-making process or program development process for that initiative. By contrast, if the GitHub Sponsors Product Manager wanted to attend the call, I would not find that person to be a reasonable inclusion to the group, because the primary motivation would be to gain insights that empower the development of the GitHub Sponsors product. I would, however, find it reasonable to offer that person an ad-hoc call with the group (which is something we've been discussing for a few weeks but haven't made happen yet). If the GitHub Sponsors Product Manager was also developing a direct funding initiative that wrote checks to spend GitHubs money on supporting GitHub's open source dependencies, then I'd probably have to make a call that one of didn't agree with.

To @ShaneCurcuru's point, however, my concern is with the word "Truly." We can, to a degree, assess the Why of a funding program (is this a pure marketing play or are they trying to do the right thing?), but the deeper we try to suss out someone's True motivations. I don't know that I want any of us to be the adjudicator of motivations on that level.

At the end of the day, what I want is a working group of people who are here to develop, improve, deploy, and share frameworks that they're using to making decisions about how to flow corporate dollars into the open source ecosystem, with a strong bias toward work that goes beyond foundation funding. If the people who show up for the calls are contributing to that effort, or learning from that effort how to implement frameworks in their own organizations, then I'd err on the side of including them.

File: index.html [hunk 1]
```diff
@@ -52,9 +52,19 @@ <h2>Where?</h2>
...
+            to participate:
           </p>
-
+          <ol>
+            <li>
+              Participants actively work within their company to adopt and
+              refine models that help unlock funding for Open Source projects.
+            </li>
+            <li>
+              Participants do this work agnostic of their company's own product
```
Comment (DuaneOBrien) [inline index.html]: Proposed replacement text:

"Participants are not involved in the development of funding platforms or funding-related products."

Comment (DuaneOBrien) [reply index.html]: @hartwork does this text read more clearly?

Comment (chadwhitacre) [inline index.html]: Great case for GitHub's inline suggestion feature. ☺️ 

```suggestion
              Participants are not involved in the development of funding platforms or funding-related products.
```

Comment (chadwhitacre) [reply index.html]: (lgtm, btw)

Comment (hartwork) [reply index.html]: @DuaneOBrien with me definitely.

Comment (chadwhitacre) [reply index.html]: Updated in 8352d0f20ea4770ca26d74a5824d36ee6ed15289...cb28cdfcde748ea30f42c4aca0dfd3a6f7689dc2. 👍

**Pull request closed without merging**